### PR TITLE
Prep custom fields integration

### PR DESCRIPTION
**Pull request by timosville**
Prep work for custom fields integration
https://trello.com/c/EvMP6l8x/461-translation-custom-fields

### CODE REVIEW

File: src/Draft/List/FlatList.vue [hunk 1]
```diff
@@ -1,33 +1,32 @@
 <template>
-  <ul class="my-8 space-y-8 bg-transparent">
-    <li v-for="(_listItem, index) in listItems" :key="index">
+  <ul
+    v-for="(_listItem, index) in listItems"
```
Comment (JannieT) [inline src/Draft/List/FlatList.vue:3]: interesting semantics here. We are changing from one `ul` containing n `li` list items to n `ul` items each containing 1 `li` item. Is this necessary?

File: src/store/widgets.ts
Comment (JannieT) [inline src/store/widgets.ts:1]: looks like we are not using sizeOfItems yet. Shall we remove them until we need them?

**Review by JannieT — CHANGES_REQUESTED** [review]
Good work! Looks like there are still some left-overs from your advanced attempt.

File: src/Draft/List/FlatList.vue (hunk 1 above)
Comment (timosville) [reply src/Draft/List/FlatList.vue:3]: I see the point. Good catch.

**Review by JannieT — APPROVED** [review]

**Pull request merged**